include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -15,11 +15,11 @@ class Device {
     VkDevice m_device;
     VkPhysicalDevice m_graphics_card;
     VmaAllocator m_allocator{VK_NULL_HANDLE};
-    std::string m_gpu_name;
+    std::string m_gpu_name{""};
 
-    VkQueue m_graphics_queue;
-    VkQueue m_present_queue;
-    VkQueue m_transfer_queue;
+    VkQueue m_graphics_queue{nullptr};
+    VkQueue m_present_queue{nullptr};
+    VkQueue m_transfer_queue{nullptr};
```

Comment:
There are other parts of the code where `VK_NULL_HANDLE` wasn't used in the first place but `nullptr` was used later!